transforms/ember-object/__testfixtures__/runtime.output.js

```diff
@@ -1,9 +1,9 @@
 import classic from 'ember-classic-decorator';
 import { off, unobserves } from '@ember-decorators/object';
 import { action, computed } from '@ember/object';
+import { service } from '@ember/service';
```

Comment:
This test was already passing because the service we inject here is in the mock telemetry.